python/revng/cli/download_pdb.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import os
+import shutil
+import sys
+
+import pefile
+import requests
+
+from .commands_registry import Command, Options, commands_registry
+from .support import log_error
+
+
+def log(message):
+    sys.stderr.write(message + "\n")
+
+
+def download_pdb_file(url):
+    local_filename = url.split("/")[-1]
+    local_filename = os.path.join("./", local_filename)
+    with requests.get(url, stream=True) as r, open(local_filename, "wb") as f:
+        if r.status_code == 200:
+            shutil.copyfileobj(r.raw, f)
+        else:
+            log("URL was not found.")
+
+    return local_filename
+
+
+class DownloadPDBCommand(Command):
+    def __init__(self):
+        super().__init__(("model", "download-pdb"), "Download missing PDBs for CRTs.")
+
+    def register_arguments(self, parser):
+        parser.add_argument("input", help="The input file.")
+
+    def run(self, options: Options):
+        args = options.parsed_args
+
+        log("Downloading PDB from Microsoft server...")
+        log("Input File: " + args.input)
+        log("")
```

Comment:
These messages are not super useful. I'd only communicate with the user if there's a need for it.
I like tools that do what they're supposed to silently.

If you think these and other messages are useful, consider a `--verbose` flag.